src/library/Uno.Material/Styles/Application/Colors.xaml

```diff
@@ -22,6 +22,13 @@
 			<Color x:Key="SurfaceColor">#FFFFFF</Color>
 			<Color x:Key="ErrorColor">#F85977</Color>
 			<Color x:Key="OnPrimaryColor">#FFFFFF</Color>
+			<Color x:Key="OnPrimaryMediumColor">#4AFFFFFF</Color>
```

Comment:
Je viens juste de les enlever -_-

Reply:
What do I use for SelectControlToggleOnBackgroundColor then?